src/Service.Host/client/src/components/StockTriggerLevelsUtility/StockTriggerLevelsUtility.js

```diff
@@ -371,7 +372,8 @@ StockTriggerLevelsUtility.propTypes = {
             triggerLevel: PropTypes.number,
             maxCapacity: PropTypes.number,
             palletNumber: PropTypes.number,
-            kanbanSize: PropTypes.number
+            kanbanSize: PropTypes.number,
+            storageLocation: PropTypes.arrayOf(PropTypes.shape({}))
```

Comment:
should this be arrayOf? just cos the lack of plural name 'storageLocation'  implies there's only one

Reply:
Mmm It's because it comes back with the following payload. A StorageLocation is made up of several parts. Technically it's more of a struct sort of idea. 

>  payload: {
    data: [
      {
        id: 4357485,
        partNumber: 'CAP 427',
        locationId: 15253,
        triggerLevel: 0,
        maxCapacity: 0,
        palletNumber: null,
        kanbanSize: 1,
        storageLocation: {
          locationCode: 'E-KIT-450/572',
          id: 15253,
          description: 'PCAS 450/572/573/WS',
          defaultStockPool: null
        },

Reply:
Is there a more appropriate prop type I should use rather than array?

Reply:
Ah wait it's just  storageLocation: PropTypes.shape({}) doiiii

Reply:
The shape is specified already.